src/Utilities/Compiler/Extensions/ISymbolExtensions.cs

```diff
@@ -153,7 +153,7 @@ public static ImmutableArray<IParameterSymbol> GetParameters(this ISymbol? symbo
             {
                 IMethodSymbol m => m.Parameters,
                 IPropertySymbol p => p.Parameters,
-                _ => ImmutableArray.Create<IParameterSymbol>()
+                _ => ImmutableArray<IParameterSymbol>.Empty
```

Comment:
If you're changing this line, might as well fix the missing `,` at the same time

```suggestion
                _ => ImmutableArray<IParameterSymbol>.Empty,
```